src/string/immutable.zig

```diff
@@ -1119,6 +1125,16 @@ pub fn index(self: string, str: string) i32 {
     }
 }
 
+/// Returns a substring starting at `start` up to the end of the string.
+/// If `start` is greater than the string's length, returns an empty string.
+pub fn drop(self: string, start: usize) string {
+    return self[@min(start, self.len)..];
+}
+
+pub fn dropMut(self: []u8, start: usize) []u8 {
```

Comment:
This might be a good case for a single function that can take a `[]u8` or `[]const u8` via `anytype`.